spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/spring-cloud-gcp-pubsub-bus-config-sample-client/src/main/resources/application.properties

```diff
@@ -1,3 +1,6 @@
+spring.config.use-legacy-processing=true
```

Comment:
I think you should be able to remove this with the addition of the starter.

`spring.config.use-legacy-processing=true` doesn't have any effect if it is added in `application.properties` (at least this is what I understood from Spencer's explanation).